apps/test/solutions/applab/ec_simple.js

```diff
@@ -26,6 +26,103 @@ module.exports = {
       },
     },
 
+    // TODO - Data blocks
+
+    // These exercise all of the blocks in Control category
+    // It does not validate that they behave correctly, just that we don't end
+    // up with an errors
```

Comment:
I think this might be why I just put a TODO here to start rather than just doing the work.